### ISSUE TITLE
Unable to press 'Accept' button while in TalkBack mode

### ISSUE THREAD

**Issue by kayaelle** [digitalcredentials/learner-credential-wallet#396]
From Pivotal list

Labels: none

status: unscheduled

created Jan 11, 2022

by Dmitri

Description: (Tested on Android, Google Pixel 2 hardware)
To replicate:
1. Accessibility > enable TalkBack
2. Add Credential > Scan QR Code
3. Scan a sample QR code containing credentials, goes to 'Available Credentials' confirmation screen
4. Click on 'Accept' button. 
Expected: Accept button gets highlighted for TalkBack (so the user can double-tap to activate it).
Actual: The whole credential gets highlighted for TalkBack. Double-tapping the Accept button goes to Credential Preview screen.

**Comment by alexfigtree** [comment]
No longer occurring. Double tapping on "Accept" screen now accepts credential, adds to wallet, and navigates to Available Credentials screen. Closing ticket.
